tests/api_resources/PhoneNumberTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Telnyx;
+
+class PhoneNumberTest extends TestCase
+{
+    const TEST_RESOURCE_ID = '123';
+
+
+    public function testIsListable()
+    {
+        $this->expectsRequest(
+            'get',
+            '/v2/phone_numbers'
+        );
+        $resources = PhoneNumber::all();
+        $this->assertInstanceOf(\Telnyx\Collection::class, $resources);
+        $this->assertInstanceOf(\Telnyx\PhoneNumber::class, $resources[0]);
+    }
+
+    public function testIsUpdatable()
+    {
+        $this->expectsRequest(
+            'patch',
+            '/v2/phone_numbers/' . urlencode(self::TEST_RESOURCE_ID)
+        );
+        $resource = PhoneNumber::update(self::TEST_RESOURCE_ID, [
+            "name" => "Test",
+        ]);
+        $this->assertInstanceOf(\Telnyx\PhoneNumber::class, $resource);
+    }
+
+    /*
+    public function testIsRetrievable()
```

Comment:
How do I get the voice and messaging settings for a single phone number? I see the collection methods above via `\Telnyx\PhoneNumber\Voice::All();`, I can't seem to do `\Telnyx\PhoneNumber\Voice::Retrieve('UUID');`

Reply:
This is similar to how the Ruby version does it.

$x = PhoneNumber::retrieve("1288819030748038942");
$x->voice();

And also 

$x = PhoneNumber::retrieve("1288819030748038942");
$x->messaging();

Reply:
Works like a charm